src/coreclr/nativeaot/Runtime/eventpipeinternal.cpp

```diff
@@ -100,12 +109,14 @@ EXTERN_C NATIVEAOT_API void __cdecl RhEventPipeInternal_Disable(uint64_t session
     ep_disable(sessionID);
 }
 
+EventPipeProvider * create_provider(const WCHAR*, EventPipeCallback, void* pCallbackContext = nullptr);
```

Comment:
Can we either just inline what this does directly in RhEventPipeInternal_CreateProvider (seems simple enough) or call out that this is from generated code?